src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -1185,6 +1185,15 @@ module Types = struct
                  ~args:Arg.[]
                  ~resolve:(fun _ { account; _ } ->
                    account.Account.Poly.snapp_uri)
+             ; field "snappState"
+                 ~typ:(list @@ non_null string)
+                 ~doc:"The snapp state associated with this account"
```

Comment:
can you describe this more carefully -- this refers to the 8 field elements right? Can you say something like "the 8 field elements encoded as bignum strings comprising the snapp state associated with this account"?